### PR TITLE
UI desing for the main activity

### PR DESCRIPTION
**Pull request by MarcosCGlobant**
UI desing for the calculator on Kotlin, accepting feedback on desing choices

Screenshot_1584384029

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,18 +1,204 @@
...
+        android:id="@+id/activity_main"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        android:background="@color/main_background"
+        android:orientation="vertical">
```
Comment (lucianoeli) [inline app/src/main/res/layout/activity_main.xml]: dont need this on a constrait layout

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,18 +1,204 @@
...
+            android:text="@string/button_one"
+            app:layout_constraintBottom_toTopOf="@id/button_main_number_zero"
+            app:layout_constraintEnd_toStartOf="@id/button_main_number_two"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/button_main_number_four" />
+
+        <Button
+            android:id="@+id/button_main_number_two"
+            style="@style/numberButtonStyle"
+            android:text="@string/button_two"
```
Comment (mauroLonghi) [inline app/src/main/res/layout/activity_main.xml]: Pls , rename the string resouces with another more representative.

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (MarcosCGlobant) [reply app/src/main/res/layout/activity_main.xml]: done, thanks

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (MarcosCGlobant) [reply app/src/main/res/layout/activity_main.xml]: done, thanks

**Review by nicolbidone — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**